shared/chat/conversation/messages/wrapper.native.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import React, {PureComponent} from 'react'
+import {TouchableHighlight} from 'react-native'
 import {Avatar, Box, Icon, Text} from '../../../common-adapters'
```

Comment:
same as above, switch to common-adapters/index.native and pull in NativeTouchableHighlight